src/main.hpp

```diff
@@ -623,7 +623,7 @@ extern int minotaurlevel;
 #define SPLITSCREEN 6
 
 // language stuff
-#define NUMLANGENTRIES 4100
+#define NUMLANGENTRIES 4200
```

Comment:
Everytime I think about the lang stuff I feel like it needs an overhaul. Should come after the open beta I think